addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -317,7 +317,7 @@ def _post_invoice_edi(self, invoices, test_mode=False):
     def _check_filename_is_fattura_pa(self, filename):
         return re.search("[A-Z]{2}[A-Za-z0-9]{2,28}_[A-Za-z0-9]{0,5}.((?i:xml.p7m|xml))", filename)
 
-    def _is_fattura_pa(self, filename, tree):
+    def _is_fattura_pa(self, filename, tree=None):
```

Comment:
To check that a file is in a certain format because of its filename is debatable. Other EDIs have functions like this that also check the tree structure, with similar method signatures.

This case it's even more complicated: the signed P7M format may not be a valid XML, so we first would have to anyway try and remove the signature, then check its content as an XML.

As I do have no time to implement the correct flow
1 - try and remove the signature
2 - parse the XML
3 - see if the tags relate to the FatturaPA invoice format
I've just moved things around because trying to remove the signature one time per edi_format was taking too much time.

This is "stable" so I prefer not to change the signature to completely remove the function.